### ISSUE TITLE
Support mounting other GitHub Repos

### ISSUE THREAD

**Issue by trieloff** [adobe/helix-content-proxy#265]
See https://github.com/AdobeDocs/hlxdemo/pull/1

Depends on https://github.com/adobe/helix-shared/issues/410

**Comment by rofe** [comment]
@trieloff it would be nice if this could be used in local development, too.

**Comment by trieloff** [comment]
Sounds like a simulator feature request 😃

**Comment by rofe** [comment]
Fair point. Or a request to drop the simulator entuirely and just proxy Helix Pages, sprinkled with matching local files? 😉
